h2o-py/h2o/frame.py

```diff
@@ -306,6 +306,27 @@ def filter_na_cols(self, frac=0.2):
         """
         return ExprNode("filterNACols", self, frac)._eager_scalar()
 
+    def columns_by_type(self,type="numeric"):
```

Comment:
This will raise a warning from IntelliJ saying that you're shadowing the builtin name `type`. Maybe `coltype` a better alternative?
Although the code will work as-is, it limits our ability to expand it in the future, since renaming parameters is generally a breaking change; and if I needed to write something that works with the builtin function `type` I wouldn't be able to...